src/main/java/Duke.java

```diff
@@ -6,5 +8,55 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        printGreetings();
+        storeTasks();
+    }
+
+    public static void printGreetings() {
+        String greeting = "____________________________________________________________\n"
+                + "Hello! I'm Duke\n"
+                + "What can I do for you?\n"
+                + "____________________________________________________________\n"
+                + "\n";
+
+        System.out.println(greeting);
+    }
+
+    public static void storeTasks() {
+        String line;
+        Task[] tasks = new Task[100];
+        Scanner in = new Scanner(System.in);
+        line  = in.nextLine();
+        int taskIndex = 0;
+        while (!line.equals("bye")) {
+            if (line.equals("list")) {
+                System.out.println("____________________________________________________________\n");
```

Comment:
Perhaps it would be easier to declare a String constant for the line border, so u do not need to type it out every time. 
For example, declare a private static final String MESSAGE_BOUNDARY = "    _____________________________________"